scc/git.py

```diff
@@ -3080,12 +3091,12 @@ def __call__(self, args):
                             issue.user.login,
                         ))
 
-                print("##", org.login, "(%s)" % count, "##")
-
-                for month in sorted(grouped_issues, reverse=args.descending):
-                    issues = sorted(grouped_issues[month], reverse=args.descending)
-                    print("## %s" % month)
-                    print("\n".join(sorted(issues, reverse=args.descending)))
+                if grouped_issues:
```

Comment:
Why this `if` statement? Isn't `grouped_issues` unconditionally defined in line 3081?

Reply:
This and `if issues:` in the previous block are to prevent the title from being unnecessarily printed, e.g.:
```
## IDR (0)
```